tests/Api/ApiTest.php

```diff
@@ -260,6 +262,38 @@ public function customerService()
         $this->assertNull($result[1]->getDefaultPaymentInstrument());
     }
 
+    /**
+     * @test
+     */
+    public function customerLeadSourcesService()
```

Comment:
That's for every test case? May be better to move this common test in trait instead of base class. I thin base `TestCase` is not supposed to contain tests.